pkg/loopback/loopback_test.go

```diff
@@ -0,0 +1,136 @@
+// +build linux,cgo
+
+package loopback
+
+import (
+	"flag"
+	"io/ioutil"
+	"os"
+	"strings"
+	"syscall"
+	"testing"
+
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const filePath = "/tmp/"
+
+var rootEnabled bool
+
+func init() {
+	flag.BoolVar(&rootEnabled, "test.root", false, "enable tests that require root")
```

Comment:
@dnephin It is not about whether a particular test is destructive. It is about the potential for destruction as a root-level test.

Test flags a proven pattern in Go that work much better than environment variables or build flags. Build flags often make it hard to include sets of tests and environment variables have problems with documentation. Test flags are easy to build and add and are self-documenting, since they show in the help of a test.